specification.md

```diff
@@ -674,7 +674,7 @@ a workflow with a single event state and show how data filters can be combined.
             }],
             "stateDataFilter": {
                 "input": "${ .greetings } ",
-                "output": "${ .finalCustomerGreeting }"
+                "output": "${ finalCustomerGreeting: .finalCustomerGreeting }"
```

Comment:
I don't think this is correct. 
The action data filter already pushes to "finalCustomerGreeting"
and spec says "The "toStateData" expression says that we should add/merge this selected event data to the state data ... If this property exists it should be merged, if it does not exist, one should be created."

I believe what is there now is correct.

Reply:
Yes! ^^ do that

Reply:
This is related to #632

Reply:
@ricardozanini It is, but Tristan found another issue, which is complementary to mine.

Reply:
Fixed.